community/examples/fluent-tutorial.yaml

```diff
@@ -0,0 +1,169 @@
+          16:07:46.199308305 +0000
+          @@ -6,13 +6,13 @@
+           # snpat       - the serial number pattern (ABCD-01234567)
+           
+           # Accept EULA, valid values are: {accept, decline}
+          -ACCEPT_EULA=decline
+          +ACCEPT_EULA=accept
+           
+           # Optional error behavior, valid values are: {yes, no}
+           CONTINUE_WITH_OPTIONAL_ERROR=yes
+           
+           # Install location, valid values are: {/opt/intel, filepat}
+          -PSET_INSTALL_DIR=/opt/intel
+          +PSET_INSTALL_DIR=/shared/apps/intel
+           
+           # Continue with overwrite of existing installation directory, valid values are: {yes, no}
+           CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
+          EOF1
+
+          sudo ./install.sh -s silent.cfg 
+
+        destination: /tmp/install-mpi.sh
+
+  - source: ./modules/scripts/startup-script
+    kind: terraform
+    id: startup
+    settings:
+      runners:
+      - type: shell
+        content: |
+          #!/bin/bash
+          google_mpi_tuning --hpcthroughput
+          google_mpi_tuning --nomitigation
+          yum -y install mesa-libGLU-devel libExt nmap openssl libXrandr mesa-libGL libX11-devel libSM-devel libICE-devel libXinerama libXcursor hdf5-devel dkms zlib-devel libXext-devel libGLU-devel libXt-devel libXrender-devel libXinerama-devel libpng-devel libXrandr-devel libXi-devel libXft-devel libjpeg-turbo-devel libXcursor-devel readline-devel ncurses-devel python python-devel cmake qt-devel qt-assistant mpfr-devel gmp-devel htop wget screen vim xorg-x11-drv-dummy xorg-x11-server-utils libXp-devel xorg-x11-fonts-cyrillic.noarch xterm.x86_64 motif-devel compat-libstdc++-33.x86_64 libstdc++.x86_64 libstdc++.i686 gcc-c++.x86_64 compat-libstdc++-33.i686 libstdc++-devel.x86_64 libstdc++-devel.i686 compat-gcc-34.x86_64 gtk2.i686 libXxf86vm.i686 libSM.i686 xorg-x11-fonts-ISO8859-1-75dpi.no xorg-x11-fonts-iso8859-1-75dpi.no gdm gnome-session gnome-classic-session gnome-session-xsession xorg-x11-server-Xorg xorg-x11-drv-dummy xorg-x11-fonts-Type1 xorg-x11-utils gnome-terminal gnu-free-fonts-common gnu-free-mono-fonts gnu-free-sans-fonts gnu-free-serif-fonts alsa-plugins-pulseaudio alsa-utils hdf5-devel
+
+        destination: /tmp/install-deps.sh
+      - type: shell
+        content: |
+          #!/bin/bash
+          hostname >> /home/hosts.txt
+        destination: /tmp/install-hosts.sh
+
+  - source: modules/file-system/filestore
+    kind: terraform
+    id: homefs
+    settings:
+      network_name: "default"
+      size_gb: 1024
+      filestore_tier: "BASIC_HDD"
+      local_mount: /home
+
+  - source: modules/file-system/filestore
+    kind: terraform
+    id: sharefs
+    settings:
+      network_name: "default"
+      size_gb: 1024
+      filestore_tier: "BASIC_HDD"
+      local_mount: /shared
+
+  - source: modules/compute/vm-instance
+    kind: terraform
+    id: compute_1
+    use:
+    - homefs
+    - sharefs
+    - startup
+
```

Comment:
nit: For consistency with the other modules, can this extra line be removed here and on line 156?

Reply:
Done